### DISCUSSION TITLE
cs/pl/asm/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#3] [General]
# cs/pl/asm/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/pl/asm/

**Comment by HobbitQia** [comment]
屋屋┭┮﹏┭┮
浇浇汇编

**Reply by Yusux** [reply]
浇浇汇编
~~要挂了呜呜~~

**Reply by HobbitQia** [reply]
hz浇浇
悲

**Comment by TonyCrane** [comment]
感谢 @Yusux 挑了一堆错误qwq

**Comment by RyanFcr** [comment]
图像模式的内存少了一个0 ，应该是0A000h
